imapi.c

```diff
@@ -96,7 +94,6 @@ void set_im_window(unsigned id, Rectangle rect)
     msg.win.rect = rect;
 
     int ret = write(imfd, (char *)&msg, sizeof(msg));
```

Comment:
Don't you need FCITX_UNUSED(ret) here to avoid warning? also, consider use fs::safeWrite.

Reply:
i just copied raw file from fbterm repo

Reply:
I know, but simply we have a better thing to use fcitx 5.